src/main/webapp/app/overview/course-score-calculation.service.ts

```diff
@@ -53,7 +53,11 @@ export class CourseScoreCalculationService {
                     presentationScore += participation.presentationScore ? participation.presentationScore : 0;
 
                     // programming exercises can be excluded here because their state is INITIALIZED even after the exercise is over
```

Comment:
```suggestion
                    // programming and quiz exercises can be excluded here because their state is INITIALIZED even after the exercise is over
```

Reply:
👍  5f4a6cc